Cr3d0v3r.py

```diff
@@ -1,12 +1,15 @@
 #Written by: Karim shoair - D4Vinci ( Cr3dOv3r )
 # -*- coding: utf-8 -*-
-import os,argparse,requests
+import os,argparse,requests,signal
 from getpass import getpass
 import mechanicalsoup as ms
 from Core import ispwned
 from Core.utils import *
 from Core.color import *
-
+def signal_handler(signal, frame):
+	print('\n')
```

Comment:
`B+end+'\n'` would be correct for me, but it might depend on the original console settings.